src/components/RadioButton/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import classNames from 'classnames';
+import styles from './style.module.css';
+
+
+function RadioButton({value, onChange, label, style, onBlur, name, children, className, checked, ...props}) {
+    const _className = classNames(styles.oc_radiobutton_container, styles.oc_field, className);
+    const _fieldClass = classNames(styles.oc_field_label, className)
+    const _wrapper = classNames(styles.oc_wrapper, className)
+  
+    return (
+        <div className={_wrapper}>
+            <label className={_fieldClass} style={{...style}}>
+            <input 
+                className={_className}
+                type="radio"
+                name={name}
+                onChange={onChange}
+                onBlur={onBlur}
+                value={value}
+                checked={checked}
+                {...props}
+            />
+            {children}
+    </label>
+        </div>
+    
```

Comment:
```jsx
        <div className={_wrapper}>
            <label className={_fieldClass} style={{...style}}>
                <input 
                    className={_className}
                    type="radio"
                    name={name}
                    onChange={onChange}
                    onBlur={onBlur}
                    value={value}
                    checked={checked}
                    {...props}
                />
                {children}
            </label>
        </div>
```

Reply:
lol I don't have the linting plugins and extensions on my IDE so I can't really fix all this spaces and lines 
maybe you could do that

Reply:
Can you set it up instead? You will always need it.

Reply:
sure i will...thanks

Reply:
Actually the current ESLint rules does not cover these. I'd create a PR to enforce and correct these issues